ebill_paynet/components/api.py

```diff
@@ -0,0 +1,51 @@
+# import base64
+import os
+
+from lxml import html
+from requests import Session
+from zeep import Client, Settings
+from zeep.transports import Transport
+
+WSDL_DOC = os.path.join(os.path.dirname(__file__), 'wsdl', 'DWSPayNet.wsdl')
+SSL_PROD_CERTIFICATE = os.path.join(
+    os.path.dirname(__file__), 'certificats', 'prod_services_chain.pem'
+)
+SSL_TEST_CERTIFICATE = os.path.join(
+    os.path.dirname(__file__), 'certificats', 'test_services_chain.pem'
+)
```

Comment:
Could we use [`get_resource_path`](https://github.com/odoo/odoo/blob/12.0/odoo/modules/module.py#L205) for this?

Reply:
Yes we can, changed